pkg/mimirpb/mimir.go

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
[nit] We're used to call these type of files `custom.go` because we customise the protobuf.